ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/modals/download/_cq_dialog/.content.xml

```diff
@@ -105,6 +105,31 @@
                                             fieldLabel="ZIP File Name"
                                             name="./zipFileName"
                                             required="{Boolean}true"/>
+                                    <heading
+                                        jcr:primaryType="nt:unstructured"
+                                        sling:resourceType="granite/ui/components/coral/foundation/heading"
+                                        level="{Long}4"
+                                        text="Maximum Content Size for Downloads"/>
+                                    <well
+                                        jcr:primaryType="nt:unstructured"
+                                        sling:resourceType="granite/ui/components/coral/foundation/well">
+                                        <items jcr:primaryType="nt:unstructured">
+                                            <max-content-size
+                                                jcr:primaryType="nt:unstructured"
+                                                sling:resourceType="granite/ui/components/coral/foundation/form/numberfield"
+                                                fieldDescription="Sets a maximum size limit for downloads (in bytes). Recommended to match size in AssetDownloadServlet OSGi config. Leave blank for no maximum content size."
+                                                fieldLabel="Max Content Size Limit (bytes)"
+                                                min="{Long}1"
+                                                name="./maxContentSize"/>
+                                            <max-content-size-message
+                                                jcr:primaryType="nt:unstructured"
+                                                sling:resourceType="granite/ui/components/coral/foundation/form/textarea"
```

Comment:
Should this be a textfield? I'd expect line breaks to supported for a multi-line input, which i dont think they are in the HTL.

Reply:
sure, will go for textfield